### PR TITLE
Ana/try fix extension deploy workflow node version

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: .github/workflows/extension_deployment.yml [hunk 1]
```diff
@@ -26,12 +28,3 @@ jobs:
       - name: Zip Build Files
         run: yarn build-zip
         working-directory: ${{env.WORKING_DIRECTORY}}
-      - name: Deploy to Extension Store
```
Comment (jbibla) [inline .github/workflows/extension_deployment.yml:29]: this doesn't seem like a good idea

**Comment by jbibla** [comment]
this doesn't solve the problem and removes an important part of the flow

File: .github/workflows/extension_deployment.yml (hunk 1 above)
Comment (Bitcoinera) [reply .github/workflows/extension_deployment.yml:29]: no, I was just trying xD

Comment (Bitcoinera) [reply .github/workflows/extension_deployment.yml:29]: the idea was just trying the part it fails, obviously. And then add that piece back again after fixing. We are probably going to need this actually, the new `node/setup`

**Pull request closed without merging**